tests/integration/targets/rds_instance/roles/rds_instance/tasks/test_complex.yml

```diff
@@ -97,17 +98,16 @@
     - assert:
         that:
           - not result.changed
-          - '"allocated_storage" in result.pending_modified_values or result.allocated_storage == 30'
-          - '"max_allocated_storage" in result.pending_modified_values or result.max_allocated_storage == 100'
+          - '"allocated_storage" in result.pending_modified_values or result.allocated_storage == io1_modified_allocated_storage'
+          - '"max_allocated_storage" in result.pending_modified_values or result.max_allocated_storage == 150'
           - '"port" in result.pending_modified_values or result.endpoint.port == 1150'
           - '"db_instance_class" in result.pending_modified_values or result.db_instance_class == modified_db_instance_class'
-          - '"engine_version" in result.pending_modified_values or result.engine_version == mariadb_engine_version_2'
 
-  always:
-    - name: Delete the instance
-      rds_instance:
-        id: "{{ instance_id }}"
-        state: absent
-        skip_final_snapshot: True
-        wait: false
-      ignore_errors: yes
+  # always:
```

Comment:
Why that's commented on?

Reply:
Ah did not mean to commit that ! It was when I was locally testing